urumanifest/dependencies.py

```diff
@@ -191,6 +191,7 @@ def track_client_dependency(client_path: Path, server_path: Path):
 
     def track_manifest_dependency(client_path: Path, server_path: Path, category: str, manifest_names):
         flags = ManifestFlags.installer if category in gather_installers else 0
+        flags |= ManifestFlags.bundle if client_path.suffix == ".app" else 0
```

Comment:
It would be nice to have the bundle extension in constants.py instead of as a magic string at the very minimum. On the other hand, the rest of the script (i.e. the Windows support) doesn't try to use magic strings in the filename to identify what flags need to be applied. Instead, we use the the keys from the control json files to assign a category. Then, we apply flags from the category here. So, what we should really be doing is have a dict in the control json to represent the new mac bundle. github.py will need to be updated to generate that part of the fake control json.

Reply:
I can take a look at that.

Reply:
Looks like this is still outstanding?